Source/CsDebugScript.Engine/Engine/IDebuggerEngine.cs

```diff
@@ -284,5 +284,22 @@ public interface IDebuggerEngine
         /// </summary>
         /// <param name="process">Process to Terminate.</param>
         void Terminate(Process process);
+
+        /// <summary>
+        /// Adds new breakpoint to the given process.
+        /// </summary>
+        /// <param name="process">Process.</param>
+        /// <param name="expression">Expression to be evaluated into breakpoint.</param>
+        /// <returns>New breakpoint.</returns>
+        IBreakpoint AddBreakpoint(Process process, string expression);
+
+        /// <summary>
+        /// Adds new breakpoint with assosiated action.
+        /// </summary>
+        /// <param name="process">Process.</param>
+        /// <param name="expression">Expression to be evaluated into breakpoint.</param>
+        /// <param name="action">Action to be executed when breakpoint is hit.</param>
```

Comment:
What does this expression represents? Is it compatible with other debuggers, or is it DbgEng.dll specific?

Reply:
Switching to address as input.